src/webgl/3d_primitives.js

```diff
@@ -1573,10 +1573,15 @@ p5.RendererGL.prototype.quad = function(x1, y1, z1, x2, y2, z2, x3, y3, z3, x4,
         quadGeom.faces.push([pt0, pt2, pt3]);
       }
     }
-    quadGeom
-      .computeNormals()
-      ._makeTriangleEdges()
-      ._edgesToVertices();
+    quadGeom.computeNormals()
```

Comment:
It looks like the automated tests are failing because we don't have a semicolon here, but adding this should fix it:
```suggestion
    quadGeom.computeNormals();
```